ckcp/test.sh

```diff
@@ -35,66 +48,76 @@ get_namespace() {
   echo "$KCP_NS_NAME"
 }
 
+init() {
+  KUBECONFIG_KCP="$WORK_DIR/credentials/kubeconfig/kcp/admin.kubeconfig.base"
```

Comment:
Do not use the *.base files, they're used just a couple of times during setup, and are kept around just for debug purposes. Use the *.kubeconfig files instead.

Reply:
fixed